.github/workflows/reportInstalls.yml

```diff
@@ -0,0 +1,22 @@
+name: Report Installs
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 * * * *'
+      
+jobs:
+  report-installs:
+    runs-on: ubuntu-latest
+    env: 
+      VSCE_PERSONAL_ACCESS_TOKEN: ${{ secrets.VSCE_PERSONAL_ACCESS_TOKEN }}
+      OVSX_PAT: ${{ secrets.OVSX_PAT }}
```

Comment:
secrets needed for the CLIs.

Reply:
good call...we should be using the org secret.  I just used whatever I found in our existing GHA